app/src/main/java/com/android/sample/di/module/FirebaseModule.kt

```diff
@@ -23,4 +24,10 @@ object FirebaseModule {
   fun provideFirebaseFirestore(): FirebaseFirestore {
     return FirebaseFirestore.getInstance()
   }
+
```

Comment:
added a new provideFirebaseStorage method, which provides a singleton instance of FirebaseStorage, enhancing the dependency injection setup for Firebase services.